Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -117,11 +117,8 @@ bool fontHasEitherTable(CTFontRef ctFont, unsigned tableTag1, unsigned tableTag2
 
 void Font::platformInit()
 {
-#if PLATFORM(IOS_FAMILY)
-    m_syntheticBoldOffset = m_platformData.syntheticBold() ? ceilf(m_platformData.size() / 24.0f) : 0.f;
-#else
-    m_syntheticBoldOffset = m_platformData.syntheticBold() ? 1.0f : 0.f;
-#endif
+    auto constexpr syntheticBoldScaleFactor = 36.0f;
```

Comment:
Could you write something in the commit message to explain where this number comes from? Even if it's just "I chose a number that looked nice" it at least leaves a paper trail for the magic constant.